File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file117.f90
program DRB072_taskdep1_orig_no
    use omp_lib
    implicit none

    integer :: i
    i = 0

    !$omp parallel
        !$omp single
            !$omp task depend (out:i)
            i = 1
            !$omp end task
            !$omp task depend (in:i)
            i = 2
            !$omp end task
        !$omp end single
    !$omp end parallel

    if (i/=2) then
        print*,'i is not equal to 2'
    end if
end program

File: data/StarCoder2/readable_format_HPEC/cpp/file117.cpp
#include <iostream>
#include <omp.h>

int main() {
    int i = 0;

    #pragma omp parallel
    {
        #pragma omp single
        {
            #pragma omp task depend(out:i)
            {
                i = 1;
            }
            #pragma omp task depend(in:i)
            {
                i = 2;
            }
        }
    }

    if (i!= 2) {
        std::cout << "i is not equal to 2" << std::endl;
    }

    return 0;
}
